### PR TITLE
Add video and collection metadata documentation

### PR DESCRIPTION
**Pull request by sgrshah**
- Add documentation for Videos#update as well.

### CODE REVIEW

File: source/includes/_resource.videos.md [hunk 1]
```diff
@@ -548,3 +576,156 @@ vhx.videos.files({
...
+```
+
+```php
+<?php\VHX\Videos::update();
+```
+
+> Example Request
+
+```shell
+$ curl -X POST "https://api.vhx.tv/videos/1" \
```
Comment (ksheurs) [inline source/includes/_resource.videos.md]: Shouldn't this be `PUT`?

File: source/includes/_resource.videos.md [hunk 2]
```diff
@@ -176,6 +188,19 @@ vhx.videos.create({
...
+      <td>
+        <strong class="is-block text--navy">metadata</strong>
+        <span class="text--transparent text-3">object of key value pairs</span>
+      </td>
+      <td>A set of key/value pairs that you can attach to a video object. It can be useful for
+storing additional information about the video in a structured format. Metadata keys must be
+strings. Metadata values can be strings, integers, or arrays. There are a few reserved keys that
+are auto-generated, which cannot be updated. These reserved keys are:
+<code>advertising_keywords</code>, <code>series_name</code>, <code>season_name</code>, <code>season_number</code>,
+<code>episode_number</code>, and <code>movie_name</code></p></td>
```
Comment (ksheurs) [inline source/includes/_resource.videos.md:202]: Great copy!

File: source/includes/_resource.collections.md [hunk 3]
```diff
@@ -472,19 +502,30 @@ vhx.collections.update();
 
 ```shell
 $ curl -X POST "https://api.vhx.tv/collections/1" \
-  -d description="A new description" \
+  -H "Content-Type: application/json" \
+  -d '{"description": "A new description", "metadata": {"director": "Brad Pitt", "writers": ["Foo Bar", "Bar Foo"], "release_year": 2017}}' \
```
Comment (ksheurs) [inline source/includes/_resource.collections.md:506]: Ohhh Brad Pitt, eh?!

**Review by ksheurs — COMMENTED** [review]
Looks great, @sgrshah ! Just one note on the method for updates.

File: source/includes/_resource.videos.md (hunk 2 above)
Comment (sgrshah) [reply source/includes/_resource.videos.md:202]: Thanks, used Stripe as a starting point!

File: source/includes/_resource.videos.md (hunk 1 above)
Comment (sgrshah) [reply source/includes/_resource.videos.md]: Ah, copy and paste fail. I think other #update sections might be listing `POST` as the method as well. Will fix those, nice catch!

File: source/includes/_resource.collections.md (hunk 3 above)
Comment (sgrshah) [reply source/includes/_resource.collections.md:506]: he's the best🏆

**Comment by osetinsky** [comment]
Looks good, @sgrshah ! I know we're not supporting it yet @ksheurs, but will we eventually accept nested objects as part of metadata?

**Comment by sgrshah** [comment]
@scottusrobus i'm actually going to merge this in and we can pair tomorrow on the changes you wanted to make. It will be publicly accessible for one evening...nbd just so Lionsgate can see it tomorrow.

**Pull request merged**